package.json

```diff
@@ -217,7 +217,7 @@
     "source-map-support": "^0.5.19",
     "symbol-observable": "^1.2.0",
     "tar": "^6.1.11",
-    "tinygradient": "0.4.3",
+    "tinygradient": "1.1.5",
```

Comment:
This is fine for `main`, but assuming `tinygradient` follows semver, this implies breaking changes. You noted no changes in the API, but can you summarize the `1.0.0` changes?

Reply:
The main difference was a change in build system. It looks like they also fixed their typescript definition file, so it wouldn't seem like it was exporting functions that it wouldn't: https://github.com/mistic100/tinygradient/compare/0.4.3...1.0.0#diff-1c4c0cc198af965a5be46c21d5e65adf184698513a26f7c0ba54e8702a71de66

It doesn't seem like they were following semver to a T